manifests/WiXToolset/WiXToolset/3.11.2.yaml

```diff
@@ -0,0 +1,17 @@
+Id: WiXToolset.WiXToolset
+Version: 3.11.2
+Name: WiX Toolset
+Publisher: WiX Toolset
```

Comment:
When installed, the publisher shows as ".NET Foundation"

Reply:
I was unsure about this one. I changed it.